apps/status-api/src/controllers/OracleStatusController.ts

```diff
@@ -31,12 +31,12 @@ export class OracleStatusController {
    * To provide the status of each ticker determined by the number of oracles responded given the ticker id
    *
    * @return {Promise<OracleStatus>}
-   * @param token
-   * @param currency
+   * @param pair
    */
-  @Get('/ticker/:token-:currency')
-  async getOracleRespondStatus (@Param('token') token: string, @Param('currency') currency: string): Promise<{ status: OracleStatus }> {
-    return await this.cachedGet(`oracle-${token}-${currency}`, async () => {
+  @Get('/ticker/:pair')
+  async getOracleRespondStatus (@Param('pair') pair: string): Promise<{ status: OracleStatus }> {
+    const [token, currency] = pair.split('-')
+    return await this.cachedGet(`oracle-${token}`, async () => {
```

Comment:
```suggestion
    return await this.cachedGet(`oracle-${token}-${currency}`, async () => {
```
Are we missing the `${currency}` here or was it removed on purpose?

Reply:
Ah, I wanted to replace it with `pair` but it was missed

Reply:
```suggestion
    return await this.cachedGet(`oracle-${pair}`, async () => {
```